genai/count_tokens/counttoken_compute_with_txt.py

```diff
@@ -13,8 +13,8 @@
 # limitations under the License.
 
 
-def compute_tokens() -> int:
-    # [START googlegenaisdk_count_tokens_compute_with_txt]
+def compute_tokens_example() -> int:
+    # [START googlegenaisdk_counttoken_compute_with_txt]
```

Comment:
These region tags are already being used in the live docs. I used different ones to make it easier to read.

Reply:
Holt, I have some automation work with region tags and these new changes break the pipelines. Also, region tags are not for readability of users. How do you want to proceed?

Reply:
I will add the new & old tags for now.